pmpro-add-name-to-checkout.php

```diff
@@ -74,7 +76,7 @@ function pmproan2c_pmpro_registration_checks()
 	}
 	else
 	{
-		$pmpro_msg = "The first and last name fields are required.";
+		$pmpro_msg = __("The first and last name fields are required.", "pmpro-add-name-to-checkout");
```

Comment:
Better be consistent and use `_e` here as well.

Reply:
The purpose of line 79 is to assign the error message to the $pmpro_msg global, not immediately send it to the output buffer (which is what `_e()` does).

Reply:
Thanks, @eighty20results, my bad, I thought they were synonym functions.